envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/assertions/AwaitAssertions.kt

```diff
@@ -4,10 +4,10 @@ import org.assertj.core.api.Assertions
 import org.awaitility.Awaitility
 import java.time.Duration
 
-fun <T> untilAsserted(wait: Duration = Duration.ofSeconds(90), fn: () -> (T)): T {
+fun <T> untilAsserted(poll: Duration = Duration.ofMillis(500), wait: Duration = Duration.ofSeconds(90), fn: () -> (T)): T {
```

Comment:
I would format this file to have method args in seperate lines

Reply:
Fixed.